prow/config.yaml

```diff
@@ -4904,6 +4904,49 @@ periodics:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  - name: ci-kubernetes-e2e-kubeadm-gce-ipvs
+    agent: kubernetes
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/e2e-kubeadm:v20171120-fd7d8854
+        args:
+        - "--repo=k8s.io/kubernetes=master"
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
```

Comment:
as far as I can tell this is a CI job that does not do any sort of build, it shouldn't mount the cache volume or set the port as we can only have one of these per node and it doesn't really need it.

Reply:
Sorry, I'm not very familiar with this part, Should I delete volume `cache-ssd` and `ports`?

Reply:
no worries :-)
yes both of those, and the --git-cache arg here.

Reply:
Thanks, am I doing right?